pkg/plugins/backendplugin/grpcplugin/grpc_plugin.go

```diff
@@ -40,10 +40,6 @@ func newPlugin(descriptor PluginDescriptor) backendplugin.PluginFactoryFunc {
 	}
 }
 
-func (p *grpcPlugin) CanHandleDataQueries() bool {
```

Comment:
how about the datasource_plugin.go, I saw there is still the implementation of CanHandleDataQueries() over there.

Reply:
See comment https://github.com/grafana/grafana/pull/32278#issuecomment-805793575